spring-cloud-kubernetes-commons/src/main/java/org/springframework/cloud/kubernetes/commons/config/ConfigMapPropertySource.java

```diff
@@ -54,10 +51,6 @@ public ConfigMapPropertySource(SourceData sourceData) {
 		super(sourceData.sourceName(), sourceData.sourceData());
 	}
 
-	protected static String getSourceName(String applicationName, String namespace) {
```

Comment:
moved two methods from this class to `ConfigUtils`. I had them here because they were used only for config maps - so it made sense, now they are reused in secrets also, so it made sense to move them to `ConfigUtils`